masq/src/notifications/connection_change_notification.rs

```diff
@@ -15,10 +15,13 @@ impl ConnectionChangeNotification {
     ) {
         let _lock = term_interface.lock();
         let output_string = match response.stage {
-            UiConnectionChangeStage::ConnectedToNeighbor => {
+            UiConnectionStage::NotConnected => {
+                todo!("Panic shouldn't happen after GH-623.")
```

Comment:
Can this message be clarified? When I first read it, I thought it meant that the panic happens frequently now, but after GH-623 was played, this section of code wouldn't be hit. Turns out the actual meaning is almost precisely opposite.